modules/nw-nwop-modifying-config-running.adoc

```diff
@@ -0,0 +1,75 @@
+// Module filename: nw-nwop-modifying-config-running.adoc.adoc
```

Comment:
This needs to be in a separate assembly. It's a different user story. In the new assembly, I'd reference the "installing {product-title} on AWS with network configurations" assembly as a prereq.